src/main/java/examples/MetricsExamples.java

```diff
@@ -30,10 +30,12 @@
 public class MetricsExamples {
```

Comment:
yes I wanted to validate first the core changes and refactoring, will do soon with cleanup and publish docs in stack page